README.md

```diff
@@ -52,216 +18,126 @@ microservice architectures efficiently.
-
-#### WSO2 API Microgateway toolkit commands
-
-Following are the set of commands included within the WSO2 API Microgateway.
-
-Note: Before you execute any of the commands below you need to add the path to the <micro-gw-home>/bin directory to the PATH environment variable. Ex: /home/dev/wso2am-micro-gw/bin
+Note: Before you execute any of the commands below you need to add the path to the `<CLI_HOME` directory to the PATH environment variable. Ex: /home/dev/wso2am-micro-gw/bin
 
 ##### Init
 
-`$ micro-gw init <project_name>`
-
-The "micro-gw init" command is used to initialize a project structure with artifacts required in generating a WSO2 API Microgateway distribution. This will create a **api_definitions**  directory.
-
-* **api_defintions** - API developer should copy all the open API definitions of microservices inside this directory
+`$ apictl init <project_name> --oas <filePathToOpenAPI_or_openAPIUrl`
 
-If the project already exists, a warning will be prompted requesting permission to override existing project.
+The "apictl init" command is used to initialize a project structure with artifacts required to deploy API in Microgateway. This will create a **api_definitions**  directory.
 
-Execute `micro-gw help init` to get more detailed information regarding the setup command.
+Execute `apictl help init` to get more detailed information regarding the setup command.
 
 Example
 
-
-    $ micro-gw init petstore
+    $ apictl init petstore --oas https://petstore.swagger.io/v2/swagger.json
 
 Let's see how we can expose the [petstore swagger](samples/petstore_swagger3.yaml) using the micro-gw.
 
-Let's add the basic microgateway Open API extension to the petstore OAS file.
+##### Deploy
 
+`$ apictl mg deploy --host <url_Of_ControlPlane> --file <Zipped_project_initiated_from_apictl>  --username <Username> --password <Password>`
 
+Upon execution of this command, CLI tool deploy the API described with open API in the Microgateway.
 ```
-x-wso2-basePath: /petstore/v1
-x-wso2-production-endpoints:
-  urls:
-  - https://petstore.swagger.io/v2
-
+ --host - Service url in which the Microgateway control plane is exposed.
+ --file - File path of the zip file which is the comresseed  project intitiated from apictl tool.
+ --username - A valid username in order to communicate with the control plane (ex: admin)
+ --password - The password of the user.
 ```
-
-Sample for petstore OAS file with two resources and extensions can be found [here](samples/petstore_basic.yaml)
-
-##### Build
-
-`$ micro-gw build <project_name>`
-
-Upon execution of this command, the WSO2 API Microgateway CLI tool will build the executable file for the specified project.
-
-Execute `micro-gw help build` to get more detailed information regarding the build command.
-
 Example
 
-	$ micro-gw build petstore
+	$ apictl mg deploy --host https://localhost:9095 --file petstore.zip  --username admin --password admin
```

Comment:
Either -k option needs to be mentioned or else we should advise on how to add the cert.